app/brave_settings_strings.grdp

```diff
@@ -1440,6 +1440,17 @@ Leo. This action cannot be undone.
     Data and chat history
   </message>
 
+  <!-- Settings / Playlist -->
+  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="A text for setting to en/disable Playlist feature">
```

Comment:
Mega nit: Text is a weird word - it doesn't need `A` in front of it (couldn't tell you why :laughing:).

You can write 
```suggestion
  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="Text for the setting to en/disable Playlist feature">
```

or 
```suggestion
  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="The text for the setting to en/disable Playlist feature">
```

Reply:
Oh, "text" is UC in this context! Thanks!